tracekit.js

```diff
@@ -1230,3 +1230,5 @@ if (!TraceKit.linesOfContext || TraceKit.linesOfContext < 1) {
 window.TraceKit = TraceKit;
 
 }(typeof window !== 'undefined' ? window : global));
+
+module.exports = TraceKit;
```

Comment:
Can you look into moving us to the umd format. I'm not comfortable with this change. module.exports may not exist and we have to support browsers back to ie6.